src/main/groovy/com/hierynomus/gradle/plugins/jython/tasks/DownloadJythonDeps.groovy

```diff
@@ -41,33 +42,58 @@ class DownloadJythonDeps extends DefaultTask {
         project.configurations.getByName(configuration).allDependencies.withType(ExternalModuleDependency.class)*.each { d ->
             String name = d.name
             logger.lifecycle("Downloading Jython library: $name with version ${d.version}")
+            logger.debug("BLARGH")
 
             def acceptClosure = getAcceptClosure(d)
 
             for (String repository : extension.sourceRepositories) {
-                def engine = new SimpleTemplateEngine()
-                def template = engine.createTemplate(repository)
-                def repo = template.make(['dep': d])
-                logger.lifecycle("Trying: $repo")
-                boolean found = false
+                def releaseUrl = getReleaseUrl(repository, d)
+                if(releaseUrl) {
```

Comment:
I can see that you use a non-IDE :) please add a space between the `if` and the `(`